bumble/hci.py

```diff
@@ -4397,7 +4397,7 @@ def from_bytes(packet: bytes) -> HCI_Event:
         if len(parameters) != length:
             raise ValueError('invalid packet length')
 
-        cls: Type[HCI_Event | HCI_LE_Meta_Event] | None
+        cls: Any
```

Comment:
`cls` is a local variable here, so it's not really necessary to provide a type hint here(still we need a placeholder otherwise mypy will be unhappy to the reassignment).